packages/js-sdk/src/datastore/cachestore.ts

```diff
@@ -145,9 +146,9 @@ export class CacheStore {
     return stream;
   }
 
-  async create(doc: any, options: any = {}) {
+  async createOne(doc: any, options: any = {}) {
```

Comment:
Could annotate `createOne` and `createMany` with a `private` access modifier. Or maybe `protected` makes more sense, not sure what the inheritance model is nowadays. Think it also makes sense to keep the "underscore for non-public methods" since the transpiled code has no indication for end-users.